src/modules/fancyzones/lib/FancyZones.h

```diff
@@ -81,9 +81,13 @@ interface __declspec(uuid("{5C8D99D6-34B2-4F4A-A8E5-7483F6869775}")) IZoneWindow
      */
     IFACEMETHOD_(int, GetZoneHighlightOpacity)() = 0;
     /**
-     * @returns Bool indicating if dragged window should be transparrent 
+     * @returns Boolean indicating if dragged window should be transparrent.
      */
-    IFACEMETHOD_(bool, isMakeDraggedWindowTransparentActive) () = 0;
+    IFACEMETHOD_(bool, IsMakeDraggedWindowTransparentActive) () = 0;
```

Comment:
IsMakeDraggedWindowTransparentActive has still the uppercase I from your previous commit